cfme/tests/infrastructure/test_host_drift_analysis.py

```diff
@@ -67,13 +66,16 @@ def test_host_drift_analysis(request, setup_provider, provider, host, soft_asser
         def test_host_remove_creds():
             test_host.update(
                 updates={
-                    'credentials': host.Host.Credential(
+                    'credentials': host_obj.Host.Credential(
```

Comment:
I'm not quite sure what you mean here Ievgen.

`cfme.infrastructure.host` is imported as `host_obj` (terrible name, its a class).  The function here is to remove the credentials for the `test_host` instance by updating with an empty `Credential` instance.  I used the `cfme.infrastructure.host.Host.Credential()` object as the value for the credential.

This shouldn't remove class credentials, just this instance's credentials - or am I missing something here?

When testing without this change, I saw exceptions due to host.Host.Credential being an invalid reference.